src/test/java/io/vertx/ext/asyncsql/MySQLClientTest.java

```diff
@@ -0,0 +1,71 @@
+package io.vertx.ext.asyncsql;
+
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.sql.ResultSet;
+import io.vertx.ext.unit.Async;
+import io.vertx.ext.unit.TestContext;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.List;
+
+public class MySQLClientTest extends SQLTestBase {
+
+
+  @Before
+  public void init() {
+    client = MySQLClient.createNonShared(vertx,
+        new JsonObject()
+            .put("host", System.getProperty("db.host", "localhost"))
+    );
+  }
+
+  // Date test is implementation-specific as MySQL is shrinking the end of the time (using 000).
```

Comment:
Wouldn't it make sense to the `testDateValueSelection` test inside `SQLTestBase` and have abstract `expectedTime/insertedTime` variables in it that we override in the MySQL / PostgreSQL classes?